thirdparty/build-thirdparty.sh

```diff
@@ -995,8 +995,11 @@ build_jansson() {
     mkdir -p build
     cd build
     $CMAKE_CMD .. -DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR} -DCMAKE_INSTALL_LIBDIR=lib
+    OLD_BUILD_SYSTEM=${BUILD_SYSTEM}
```

Comment:
no need to save to `$OLD_BUILD_SYSTEM`, directly use `make` in following two lines.